cmd/telegraf/main.go

```diff
@@ -118,7 +119,10 @@ func runApp(args []string, outputBuffer io.Writer, pprof Server, c TelegrafConfi
 
 	// This function is used when Telegraf is run with only flags
 	action := func(cCtx *cli.Context) error {
-		logger.SetupLogging(logger.LogConfig{})
+		err := logger.SetupLogging(logger.LogConfig{})
+		if err != nil {
+			return fmt.Errorf("E! %w", err)
+		}
```

Comment:
I already wonder what errors this is going to expose that users have not seen :)

Reply:
Errors regarding `close` problems for previous logger when new logger will be set up using this method...
Is that ok? Or should we swallow this error and not return it?